CONTRIBUTING.md

```diff
@@ -31,3 +32,52 @@ The most important thing to include in your pull request are *tests* - please wr
 Starting December 1, 2017, all contributions will be assumed to be released under a dual license - the [Apache 2.0 License](https://www.apache.org/licenses/LICENSE-2.0) and the [3-Clause BSD License](https://opensource.org/licenses/BSD-3-Clause) unless otherwise specified in the pull request.
 
 All contributions before December 1, 2017 except those explicitly relicensed, are only under the 3-clause BSD license.
+
+## Building and releasing
+
+When you get the source, it does not contain the internal zoneinfo
+database. To get (and update) the database, run the updatezinfo.py script. Make sure
+that the zic command is in your path, and that you have network connectivity
+to get the latest timezone information from IANA, or from [our mirror of the
+IANA database](https://dateutil.github.io/tzdata/).
+
+Starting with version 2.4.1, all source and binary distributions will be signed
+by a PGP key that has, at the very least, been signed by the key which made the
+previous release. A table of release signing keys can be found below:
+
+Releases | Signing key fingerprint
+-------- | ---------------------------------------------------
+2.4.1-   | [6B49 ACBA DCF6 BD1C A206 67AB CD54 FCE3 D964 BEFB]
+
+[6B49 ACBA DCF6 BD1C A206 67AB CD54 FCE3 D964 BEFB]: https://pgp.mit.edu/pks/lookup?op=vindex&search=0xCD54FCE3D964BEFB
+
+## Development Setup
+
+Install the the dependencies for running the test suite using `pip` or `conda`.
+
+### pip
+
+Run the following commands to create a [virtual environment](https://virtualenv.pypa.io) with all dependencies installed:
+
+    python -m virtualenv .venv       # Create virtual environment in .venv directory
+    . .venv/bin/activate             # Activate the virtual environment
+    pip install -r requirements.txt  # Install the dependencies
+
+### conda
+
+Run the following commands to create a [conda environment](https://conda.io) with all dependencies installed:
+
+    conda create -n dateutil python=3.6  # Create a conda environment using Python 3.6
```

Comment:
Why enforce python 3.6? As long as you're using a supported version I think it's fine